.gitignore

```diff
@@ -14,34 +14,6 @@
 *.idx
 *.ind
 
-hyperSpec/NAMESPACE
```

Comment:
NAMESPACE and man/*.Rd are auto-generated by `roxygenize()` and they are therefore redundant, i.e. the primary source is not there but in the Roxygen comments.

Can they stay ignored and instead be generated also during the actions?

Reply:
At some point, there were problems with `pkgdown` and ignored "man/*.Rd". I will check on a separate branch if these files can stay Git-ignored.

Reply:
1. `man/*.Rd` can be safely git-ignored: [R-CMD-check](https://github.com/GegznaV/hyperSpec/actions/runs/135355859) and [pkgdown](https://github.com/GegznaV/hyperSpec/runs/770795731?check_suite_focus=true) building are successful.
2. Unfortunately, if both NAMESPACE and `vignettes/*.html` files are missing, step of Roxigenation (`devtools::document()`) fails. There is the result on my local machine:
![image](https://user-images.githubusercontent.com/12725868/84638150-43773200-aeff-11ea-9467-4384cd3ac3b0.png)
And the results on [GH Actions](https://github.com/GegznaV/hyperSpec/runs/770800625?check_suite_focus=true)
And if I run `document()` function several times in console, I get an additional warning:
![image](https://user-images.githubusercontent.com/12725868/84638434-a8cb2300-aeff-11ea-84dd-d8f649e40dbc.png)
Does the last warning give any helpful information for you, @cbeleites?

    I have to run Roxygenation + building steps several times locally before the process is successful. And I did not manage to solve this issue for the GitHub Actions setup is NAMESPACE is missing. This might be a bug in `devtools::document()` or somehow related **hyperSpec** (e.g., non standard usage of **testthat**).
 **But** if NAMESPACE file is present (even incomplete), the issue disappears, [R-CMD-check passes](https://github.com/GegznaV/hyperSpec/actions/runs/135355859) (this is the same link as in point 1).

Would it be ok if we Git-ignored `man/*.Rd`  but tracked changes in NAMESPACE?